storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/introspection/StorIOColumnMeta.java

```diff
@@ -41,6 +41,16 @@ public boolean isMethod() {
         return element.getKind() == ElementKind.METHOD;
     }
 
+    public String getRealElementName() {
+        if (elementName.startsWith("get") && Character.isUpperCase(elementName.charAt(3))) {
+            return elementName.substring(3).toLowerCase();
```

Comment:
I mean names like this will be affected:
`myAwesomeName` -> `getMyAwesomeName` -> `myawesomename`
and if someone strange person will decided to add another field `myawesomename` he will have paid the price 😿